x-pack/plugins/security_solution/common/endpoint/data_loaders/index_endpoint_hosts.ts

```diff
@@ -139,12 +140,13 @@ export async function indexEndpointHostDocs({
 
     if (enrollFleet) {
       const { id: appliedPolicyId, name: appliedPolicyName } = hostMetadata.Endpoint.policy.applied;
+      const uniqueAppliedPolicyName = `${appliedPolicyName}-${uuid.v4()}`;
```

Comment:
I think ideally we do this within the data generator itself but since this is blocking us from unskipping our tests, I wanted to make the smallest changes possible for this PR.